packages/sources/finage-test/src/endpoint/http/uk-etf.ts

```diff
@@ -0,0 +1,61 @@
+import { HttpTransport } from '@chainlink/external-adapter-framework/transports'
+import { EquitiesEndpointTypes } from '../types'
+
+interface ResponseSchema {
+  symbol: string
+  price: number
+  timestamp: number
+  error?: string
+}
+
+type HttpTransportTypes = EquitiesEndpointTypes & {
+  Provider: {
+    RequestBody: never
+    ResponseBody: ResponseSchema
+  }
+}
+
+export const httpTransport = new HttpTransport<HttpTransportTypes>({
+  prepareRequests: (params, config) => {
+    return params.map((param) => {
+      const symbol = `${param.base}`.toUpperCase()
```

Comment:
Could we just do `const symbol = param.base?.toUpperCase()`?

Using a string template here will disguise type issues, e.g. if `param.base` was `undefined` this would result in the string `UNDEFINED`.